docs/advanced-features/athena.md

```diff
@@ -1,91 +1,317 @@
 # Querying package metadata with Athena
-Quilt stores package data and metadata in S3. Metadata lives in a per-package manifest file
-in a each bucket's `.quilt/` directory.
-
+Quilt stores package data and metadata in S3. Metadata lives in a per-package manifest file in each bucket's `.quilt/` directory.
```

Comment:
This change happened probably because of auto-formatting, and will be reverted with someone else auto-formatting (also, consider this https://github.com/quiltdata/quilt/pull/2987)

Reply:
Reverted